src/test/java/org/mockitousage/matchers/VarargsTest.java

```diff
@@ -304,24 +300,15 @@ public void shouldNotCaptureVarArgs_1args2captures() {
                 .isInstanceOf(ArgumentsAreDifferent.class);
     }
 
-    /**
-     * As of v2.0.0-beta.118 this test fails. Once the github issues:
-     * <ul>
-     * <li>'#584 ArgumentCaptor can't capture varargs-arrays
-     * <li>#565 ArgumentCaptor should be type aware' are fixed this test must
-     * succeed
-     * </ul>
-     */
     @Test
-    @Ignore("Blocked by github issue: #584 & #565")
```

Comment:
Does this mean this PR also fixes #565?

Reply:
in short, no.

This PR addresses #584, which is what this test tests.
#565 is not vararg related. Not sure why this test as tagged with the issue as it doesn't seem related.